sdks/java/io/jdbc/src/main/java/org/apache/beam/sdk/io/jdbc/JdbcIO.java

```diff
@@ -1971,6 +1982,7 @@ public PCollection<V> expand(PCollection<T> input) {
                       .setRetryConfiguration(getRetryConfiguration())
                       .setReturnResults(true)
                       .setBatchSize(1L)
+                      .setMaxBatchBufferingDuration(200L)
```

Comment:
Use the constant (DEFAULT_MAX_BATCH_BUFFERING_DURATION) as it is defined there

Reply:
thanks, just made that change